utilities/download_ngc_data

```diff
@@ -29,6 +29,11 @@ while [[ $# -gt 0 ]]; do
       shift # past argument
       shift # past value
       ;;
+    --dataset_name)
```

Comment:
Could we add the same in Holoscan SDK upstream for our next upcoming release? https://github.com/nvidia-holoscan/holoscan-sdk/blob/main/scripts/download_ngc_data